web/src/i18n/locales/en.yaml

```diff
@@ -133,7 +133,7 @@
     namePrefix: "Activity"
     namePrefixAndNum: "Core Activity {{number}}"
     description: 
-      title: "Key Program Administration Information"
+      title: "Activity Overview"
```

Comment:
I changed this to Activity Overview because that is a better name for this. Does this break anything that is doing "key" stuff? @brendansudol

Reply:
Good call @NAretakis